### ISSUE TITLE
Error when I add the panel to dashboard

### ISSUE THREAD

**Issue by neodani** [ilgizar/ilgizar-candlestick-panel#17]
Hi ilgizar

I can't try the panel.

Plugin Error
Unable to dynamically transpile ES module A loader plugin needs to be configured via `SystemJS.config({ transpiler: 'transpiler-module' })`. Instantiating http://localhost:3000/public/plugins/ilgizar-candlestick-panel/module.js Loading plugins/ilgizar-candlestick-panel/module

I tried to install the plugin first copying the folder of git, and in the by other hand
grafana-cli --pluginUrl https://github.com/ilgizar/ilgizar-candlestick-panel/archive/master.zip plugins install ilgizar-candlestick-panel

It doens't work me

I'm using docker-compose like

    grafana:
        image: grafana/grafana:5.1.2
        volumes:
            - ./grafana/etc:/etc/grafana
            - ./grafana/var/lib:/var/lib/grafana
            - ./grafana/var/lib/plugins:/var/lib/grafana/plugins
            - ./grafana/var/lib/provisioning:/var/lib/grafana/provisioning
            - ./grafana/var/log:/var/log/grafana
        links:
            - influxdb:influxdb
        environment:
            - GF_INSTALL_PLUGINS=grafana-clock-panel,briangann-gauge-panel,natel-plotly-panel,grafana-simple-json-datasource
        ports:
            - 3000:3000
        depends_on:
            - influxdb
        privileged: true

Any sugestion?

Thank you

**Comment by davidromeroantequera** [comment]
I have experienced this same issue. Running locally on desktop computer. Tried both installation methods as well. Plugin is not showed when doing `grafana-cli plugins ls`

**Comment by UzLA** [comment]
I have same problem: installed the plug-in globally (/var/lib/grafana/plugins/) via sudo. It appears in the list of plugins on web interface at http://localhost:3000/plugins . But when I add the candlestick panel to a dashboard, I got the following error: "
**Plugin Error
Unable to dynamically transpile ES module A loader plugin needs to be configured via `SystemJS.config({ transpiler: 'transpiler-module' })`. Instantiating http://localhost:3000/public/plugins/ilgizar-candlestick-panel/module.js Loading plugins/ilgizar-candlestick-panel/module**
"
With this it creates an empty panel on a dashboard.

**Comment by iclchan** [comment]
Hi, I had a similar problem and i realised that i made the silly mistake of downloading the zip as a clone from the repository and tried to install the project zip as a plugin. Instead, i downloaded the pre-built zip in the 'pack' folder of the repo: ilgizar-candlestick-panel.zip. I manually unzipped it and placed it in the plugins folder /usr/local/var/lib/grafana/plugins for Mac and it can load nicely.

Hope this helps and thanks for the work with the candlesticks!

**Comment by bunyk** [comment]
Yes, for me it worked when I unzipped downloaded dist folder from https://github.com/ilgizar/ilgizar-candlestick-panel/blob/master/pack/ilgizar-candlestick-panel.zip to `/usr/local/var/lib/grafana/plugins`

Probably this should be described somewhere in README.
